flaky tests: test_whitelist_actions_bypass_confirmation

```

=========================== short test summary info ============================
FAILED tests/agents/test_interactive_textual.py::test_whitelist_actions_bypass_confirmation - assert "echo 'safe'" in "Step 1/1 - RUNNING ⠸ - Cost: $0.00\nSYSTEM\nYou are a helpful assistant that can do anything.\nUSER\nYour task: Whitelist test. Please reply with a single shell command in triple backticks. To finish, the first line of the output of the shell command must be 'COMPLETE_TASK_AND_SUBMIT_FINAL_OUTPUT'."
 +  where "Step 1/1 - RUNNING ⠸ - Cost: $0.00\nSYSTEM\nYou are a helpful assistant that can do anything.\nUSER\nYour task: Whitelist test. Please reply with a single shell command in triple backticks. To finish, the first line of the output of the shell command must be 'COMPLETE_TASK_AND_SUBMIT_FINAL_OUTPUT'." = get_screen_text(TextualAgent(title='Step 1/1 - RUNNING ⠸ - Cost: $0.00', classes={'-dark-mode'}, pseudo_classes={'focus', 'dark'}))👋 This is mini-swe-agent version 1.9.1.
Your config is stored in '/home/runner/.config/mini-swe-agent/.env'
```

again:

```

Coverage XML written to file coverage.xml
=========================== short test summary info ============================
FAILED tests/agents/test_interactive_textual.py::test_whitelist_actions_bypass_confirmation - assert "echo 'safe'" in "Step 1/1 - RUNNING ⠼ - Cost: $0.00\nSYSTEM\nYou are a helpful assistant that can do anything.\nUSER\nYour task: Whitelist test. Please reply with a single shell command in triple backticks. To finish, the first line of the output of the shell command must be 'COMPLETE_TASK_AND_SUBMIT_FINAL_OUTPUT'."
 +  where "Step 1/1 - RUNNING ⠼ - Cost: $0.00\nSYSTEM\nYou are a helpful assistant that can do anything.\nUSER\nYour task: Whitelist test. Please reply with a single shell command in triple backticks. To finish, the first line of the output of the shell command must be 'COMPLETE_TASK_AND_SUBMIT_FINAL_OUTPUT'." = get_screen_text(TextualAgent(title='Step 1/1 - RUNNING ⠼ - Cost: $0.00', classes={'-dark-mode'}, pseudo_classes={'dark', 'focus'}))👋 This is mini-swe-agent version 1.16.0.
Loading global config from '/home/runner/.config/mini-swe-agent/.env'
```

--- a/tests/agents/test_interactive_textual.py
+++ b/tests/agents/test_interactive_textual.py
@@ -401,7 +401,7 @@ async def test_whitelist_actions_bypass_confirmation():
         await pilot.pause(0.2)  # Wait for UI to be ready
         threading.Thread(target=lambda: app.agent.run("Whitelist test"), daemon=True).start()
 
-        for _ in range(50):
+        for _ in range(10):
             await pilot.pause(0.1)
             if "echo 'safe'" in get_screen_text(app):
                 break

